plugins/tiddlywiki/codemirror/engine.js

```diff
@@ -13,14 +13,33 @@ Text editor engine based on a CodeMirror instance
 "use strict";
 
 var CODEMIRROR_OPTIONS = "$:/config/CodeMirror",
-	HEIGHT_VALUE_TITLE = "$:/config/TextEditor/EditorHeight/Height"
+	HEIGHT_VALUE_TITLE = "$:/config/TextEditor/EditorHeight/Height",
+	CONFIG_FILTER = "[all[shadows+tiddlers]prefix[$:/config/codemirror/]]"
+
+/*
+Apply a callback to each module of a particular type
+	moduleType: type of modules to enumerate
+	callback: function called as callback(title,moduleExports) for each module
+*/
+$tw.modules.forEachModuleOfType = function(moduleType,callback) {
+	var modules = $tw.modules.types[moduleType];
+	$tw.utils.each(modules,function(element,title) {
+		callback(title,$tw.modules.execute(title));
+	});
+};
 
```

Comment:
```
$tw.modules.forEachModuleOfType = function(moduleType,callback) {
	var modules = $tw.modules.types[moduleType];
	$tw.utils.each(modules,function(element,title) {
		callback(title,$tw.modules.execute(title));
	});
};
```

uups, this whole block should have been removed :/ I just copied it, as an "commented" example. ... but it seems I did undo the comments ..

Reply:
It's removed!